### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by eduardpredescu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,3 +92,39 @@ In order to add `<changefreq/>` or `<priority/>` to some locations you need to a
   1. Add `sitemap: false` to the front matter of that page. This excludes it from the automatic sitemap generator, then
   2. Add that url and additional properties in `sitemap_handmade.xml`
 
+# Close jobs
+In the YAML front matter of the job you want to close add:
+```
+status: closed
```
Comment (ppscvalentin) [inline README.md]:
```
job:
  status: close
```

File: README.md [hunk 2]
```diff
@@ -92,3 +92,39 @@ In order to add `<changefreq/>` or `<priority/>` to some locations you need to a
...
+  category: ''
+  thumbnail: ''
+  applyUrl: ''
+  status: closed
+```
+
+# Job/Company sorting
+In the YAML front matter of the job/company you want to sort add:
+```
+    weight: # a number representing the job/company importance(default:0 for company, default:-1 for job)(current company max:100)
```
Comment (ppscvalentin) [inline README.md]: asa e? default -1 la joburi?

File: README.md [hunk 3]
```diff
@@ -92,3 +92,39 @@ In order to add `<changefreq/>` or `<priority/>` to some locations you need to a
...
+```
+job:
+  title: Junior Frontend Developer
+  description: >
+    Platforma noastră este scrisă folosind Python și Django. Datele le analizăm utilizând tehnici de machine learning.
+  department: Departamentul Web development
+  category: ''
+  thumbnail: ''
+  applyUrl: ''
+  weight:2000
```
Comment (ppscvalentin) [inline README.md]: pune si tu spatii `weight: 2000`

File: README.md (hunk 2 above)
Comment (eduardpredescu) [inline README.md]: da

**Comment by ppscvalentin** [comment]
nu te deranjeaza ca readme-ul are:
add job/company
asset versioning
sitemap
si abia apoi
close job
sort job

**Comment by ppscvalentin** [comment]
vezi ff6783ca6c3b67d6568463eede26fcd32385b87f

**Pull request closed without merging**